### PR TITLE
Display stacked federation tiles in Pool

### PR DESCRIPTION
**Pull request by rzulian**

### CODE REVIEW

File: src/components/Commands.vue [hunk 1]
```diff
@@ -395,8 +395,9 @@ export default class Commands extends Vue {
 
 i.planet {
   &::before {
+    font-size: 25px;
     content: "\25cf";
-    filter: drop-shadow(0px 0px 3px white);
```
Comment (coyotte508) [inline src/components/Commands.vue]: why?

**Comment by coyotte508** [comment]
Can you show a screenshot of how it looks now?

**Comment by rzulian** [comment]
I've reintroduced the shadow in black. What do you think? It would be better to use a real Token though.
screen shot 2018-07-29 at 09 04 25

**Comment by rzulian** [comment]
The same in white 
screen shot 2018-07-29 at 09 07 48

**Comment by rzulian** [comment]
Mixed: 
screen shot 2018-07-29 at 09 15 11

**Comment by coyotte508** [comment]
The shadow in black is nice for the player info part. But for the faction selection buttons, at the beginning of the game, isn't the shadow in white better?

I was talking about a screenshot for the federation tiles in the pool :)

**Comment by rzulian** [comment]
Mixed black and white seems to work best

**Comment by rzulian** [comment]
screen shot 2018-07-29 at 17 35 04

**Comment by coyotte508** [comment]
Nice!

**Pull request merged**